lib/mutant.rb

```diff
@@ -281,7 +292,7 @@ class Config
       jobs:                  nil,
       matcher:               Matcher::Config::DEFAULT,
       mutation_timeout:      nil,
-      reporter:              Reporter::CLI.build(WORLD.stdout),
+      reporter:              Reporter::CliCompact.build(WORLD.stdout),
```

Comment:
This is a global change, which works for your instance. I'll have to introduce the concept of selecting a set-of reporters at the CLI/config file.